roles/openshift_cfme/files/openshift_cfme.exports

```diff
@@ -0,0 +1,3 @@
+/exports/miq-pv01 *(rw,no_root_squash,no_wdelay)
+/exports/miq-pv02 *(rw,no_root_squash,no_wdelay)
+/exports/miq-pv03 *(rw,no_root_squash,no_wdelay)
```

Comment:
Also I'm missing the steps wherein I verify the export directories exist and create them if they do not.